### DISCUSSION TITLE
Join the GitHub Copilot relevant issues

### DISCUSSION THREAD

**Discussion by darverise** [community/community#17195] [Copilot Conversations] [answered]
Request to Join the GitHub Copilot waitlist.

I've been waiting for many days. Is the application successful or unsuccessful?

Please, please give me a mandate.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
